### DISCUSSION TITLE
Playground 2.5

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/stable-diffusion-webui-forge#480] [General]
Is there away to run New Playgroundai 2.5 model on Forge. Which is only available only ComfyUI

**Comment by newxhy** [comment]
no

**Reply by Synchronos** [reply]
The SD.Next fork by vladmandic will run Playground. YMMV. SD.Next is not as stable and seems to have frequent Out of Memory kills.

**Comment by wisdombox44** [comment]
Thank You. I try on ComfyUI it works good. My main webui is Fooocus, I still play with ComfyUI even i don't like the Nodes UI. All new models works on comfy I tried Stable Cascade too. New Pxart Sigma model is really nice gives you more artistic and cinematic results. I tried on hugging face, it is available ComfyUI also but I did not set up yet. I wish all those new models works with Fooocus.

**Reply by ILikeToasters** [reply]
Works with Fooocus now.
